Sources/Util/IndexTree.swift

```diff
@@ -267,7 +267,7 @@ extension IndexTreeNode {
     /**
      * `insertBefore` inserts the given node before the given child.
      */
-    func insertBefore(newNode: Self, referenceNode: Self) throws {
+    func insertBefore(_ newNode: Self, _ referenceNode: Self) throws {
```

Comment:
How about
```swift
func insert(_ newNode: Self, before childNode: Self) throws
```

Reply:
As mentioned earlier.  I will apply the swift style to a stable version later.